src/Integration/LocalServices/SolutionRuleSetsInformationProvider.cs

```diff
@@ -58,26 +58,57 @@ public IEnumerable<RuleSetDeclaration> GetProjectRuleSetsDeclarations(Project pr
             }
         }
 
-        public string CalculateSolutionSonarQubeRuleSetFilePath(string sonarQubeProjectKey, string fileNameSuffix)
+        public string GetSolutionSonarQubeRulesFolder()
+        {
+            var projectSystem = this.serviceProvider.GetService<IProjectSystemHelper>();
+            string solutionFullPath = projectSystem.GetCurrentActiveSolution().FullName;
```

Comment:
`GetCurrentActiveSolution()` could return `null` (in the case there is no active solution). Use `GetCurrentActiveSolution()?.FullName`